torch/_inductor/pattern_matcher.py

```diff
@@ -1087,6 +1111,8 @@ def run_node(self, node: torch.fx.Node) -> Any:
                         old_node=node,
                         pass_name="Interpreter_Replacer",
                     )
+                    if "arg_kwarg_vals" in node.meta:
```

Comment:
Similarly, I'm wary of extending this too generally. If you replace one operator with a different operator, does it have the same generic input strides requirements ? I dont think we can say that generally. Also the schema could be different

Reply:
Let's say we're replacing an operator foo with a subgraph that includes the operator bar. This line isn't propagating foo's arg_kwarg_vals to bar's arg_kwarg_vals.

Instead, the sequence of events is:
- we trace out the subgraph. This subgraph's nodes may have arg_kwarg_vals in it (if proxy_tensor.py decided that those need to exist),
- we copy all nodes in subgraph to the graph the PatternMatcher is editing. When we copy the subgraph, we propagate certain meta vals over.  This line says that if the subgraph already has arg_kwarg_vals, then we should propagate it.